### ISSUE TITLE
Subscribe input missing label

### ISSUE THREAD

**Issue by andrew-t-james** [Vets-Who-Code/vets-who-code-app#214]
**Describe the bug**
Input missing label see [here](https://web.dev/label/?utm_source=lighthouse&utm_medium=devtools)

**To Reproduce**
1. Go to vetswhocode.io
2. Run lighthouse report
3. View accessibility report.

**Expected behavior**
Input should have a label.
